### PR TITLE
docs: Add Meltano installation instructions

### PR DESCRIPTION
**Pull request by edgarrmondragon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,4 +37,41 @@ pipx install git+https://github.com/MeltanoLabs/meltano-map-transform.git@v0.0.1
...
+     mappers:
+     - name: meltano-map-transformer
+       pip_url: "git+https://github.com/MeltanoLabs/meltano-map-transform.git"
+       executable: meltano-map-transform
+       mappings:
+       - name: hash_email
+         config:
+           stream_maps:
+             customers:
+               email: null # drop the PII field from RECORD and SCHEMA messages
```
Comment (aaronsteers) [inline README.md:66]: I believe `null` config values are being pruned from the config set before execution. 

This might not work as expected as of now.

Comment (aaronsteers) [reply README.md:66]: That's bizarre... 🤔

I mean, it's great but also unexpected... 😅

Comment (edgarrmondragon) [reply README.md:66]: @aaronsteers and I just confirmed that `__else__: null` also works fine: https://github.com/MeltanoLabs/meltano-map-transform/actions/runs/3200496207.

Maybe https://github.com/meltano/sdk/issues/983 is no longer true?

Comment (aaronsteers) [reply README.md:66]: @edgarrmondragon - Thanks for logging. I wonder if anyone can still repro the old issue, and if so, perhaps there is some difference in configuration or dialect. 

It's also possible with the various config fixes from Ken and Cody, we may have inadvertently fixed this issue also.

cc @visch

**Review by aaronsteers — APPROVED** [review]
Looks good! We had a tangent into the `null` behavior but since that isn't blocking as of now, I think this is ready to go.

**Pull request merged**